beacon-chain/p2p/gossip_topic_mappings.go

```diff
@@ -3,13 +3,12 @@ package p2p
 import (
 	"reflect"
 
-	"github.com/gogo/protobuf/proto"
 	pb "github.com/prysmaticlabs/prysm/proto/eth/v1alpha1"
 )
 
 // GossipTopicMappings represent the protocol ID to protobuf message type map for easy
 // lookup.
-var GossipTopicMappings = map[string]proto.Message{
```

Comment:
change this back to proto.Message, and also do the same for all the subscription handlers and validators. We only need to change to `interface{}` for rpc